tests/subsys/dfu/mcuboot/prj.conf

```diff
@@ -6,5 +6,4 @@ CONFIG_STREAM_FLASH=y
 CONFIG_IMG_MANAGER=y
 CONFIG_MCUBOOT_IMG_MANAGER=y
 CONFIG_IMG_BLOCK_BUF_SIZE=512
-CONFIG_ARM_MPU=n
```

Comment:
Won't this break this test on a lot of Arm boards?

Reply:
There is only 1 board listed in the testcase.yml file which is an nrf52840dk_nrf52840